frontend/src/components/explorer/MapViewer.vue

```diff
@@ -70,7 +70,8 @@
           <ErrorPanel :message="loadMapErrorMessage" @hideErrorPanel="loadMapErrorMessage=''" />
         </div>
         <div id="dataOverlayBar"
-             class="column is-narrow has-text-white is-unselectable is-hidden-mobile fixed-height-desktop p-1"
+             class="column is-clickable is-narrow has-text-white
```

Comment:
I'm not quite clear about the effect of `is-clickabble` here.

Reply:
It achieves the same as `cursor: pointer;` which was removed from this file.